### ISSUE TITLE
Splines and lines

### ISSUE THREAD

**Issue by SpooikypokDev** [dalboris/vpaint#69]
I think the behavior of the spline where the extremities react differently to drag and drop than the others should be removed, because it prevents someone who reshaped a spline to reshape the extremities the way they want to.

**Comment by SpooikypokDev** [comment]
No, i just understood how to work around it and i understood the purpose.

**Comment by dalboris** [comment]
Thanks for the feedback anyway! Note that I did debate for a long time this behavior of extremities. I think a good solution is to implement more "sculpt tools", and one of them should behave the same for extremities as for the interior of edges.

**Comment by SpooikypokDev** [comment]
Good idea! Shall we reopen this?
